biome.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+  "formatter": {
+    "enabled": true,
+    "indentStyle": "tab",
```

Comment:
Any preference to use tabs over spaces (4) as we do in other repos?

Reply:
None. Got confused due to `.editorconfig`. Moved back to spaces !